main/src/ca/uwaterloo/flix/language/errors/LexerError.scala

```diff
@@ -26,20 +26,20 @@ sealed trait LexerError extends CompilationMessage {
 object LexerError {
 
   /**
-   * An error raised when an unexpected character, such as €, is encountered
+   * An error raised when an unexpected character, such as €, is encountered.
    *
-   * @param s the problematic character. This is a string to handle escaped characters
-   * @param loc  the location of `char`
+   * @param s   the problematic character. This is a string to handle escaped characters.
```

Comment:
What does it mean "This is a string to handle"? Maybe simply:

```@param s the unexpected character```?

Reply:
Good suggestion. I was trying to justify using the type string instead of char for `s` since I thought that might confuse a reader. I think its unnecessary though.